### PR TITLE
fix(FormControls): add .pf-m-disabled css, document .pf-m-form-control-alt

### PR DESCRIPTION
**Pull request by michael-coker**
fix https://github.com/patternfly/patternfly-next/issues/497

### CODE REVIEW

File: src/patternfly/components/FormControls/docs/code.md [hunk 1]
```diff
@@ -8,3 +8,5 @@ Input, Textarea, and select are provided in the form controls component for use
 | Class | Applied To | Outcome |
 | -- | -- | -- |
 | `.pf-c-form-control` | `<input>`,`<textarea>`, `<select>` |  Initiates an input, textarea or select. For styling of checkboxes or Radios see the <a href="/components/Check/examples/">check component</a>. **required**  |
+| `.pf-m-form-control-alt` | `input[type="text"].pf-c-form-control` |  Applies alternative form control styles. |
+| `.pf-m-disabled` | `.pf-c-form-control` |  Applies disabled styles. |
```
Comment (matthewcarleton) [inline src/patternfly/components/FormControls/docs/code.md]: Should we explain that this class is not needed when the the disabled attribute is present?

Comment (michael-coker) [reply src/patternfly/components/FormControls/docs/code.md]: Good question. I suppose it should, just so it's clear. This state modifier is also on dropdown and the documentation just says "Modifies to display the menu item as disabled."

FWIW, it's also on buttons, but the modifier class isn't documented (it should be). The `disabled` attribute is in the accessibility table. I like the way the button accessibility table describes the state modifiers, like...

> Forces display of the active state of the button. This state is primarily for demonstration purposes and would not normally be used in lieu of the :active pseudo-class.

I think using something like that here for the disabled modifier class here (and on buttons) would work well. What do you think?

Comment (matthewcarleton) [reply src/patternfly/components/FormControls/docs/code.md]: ya I like that. I think it's good to clarify its purpose and what is typically expected.

Comment (michael-coker) [reply src/patternfly/components/FormControls/docs/code.md]: Cool, I'll open a follow-up ticket to make sure we have modifier classes on components for states where we're styling the pseudo class, and make sure that class is reflected in the documentation similarly to the state modifier classes in the button docs.

**Pull request merged**